src/app/util/attribute-storage.h

```diff
@@ -201,6 +201,12 @@ const EmberAfCluster * emberAfGetClusterByIndex(chip::EndpointId endpoint, uint8
 //
 chip::Span<const EmberAfDeviceType> emberAfDeviceTypeListFromEndpoint(chip::EndpointId endpoint, CHIP_ERROR & err);
 
+//
+// Retrieve the tag list associated with a specific endpoint.
+//
+chip::Span<const chip::app::Clusters::Descriptor::Structs::SemanticTagStruct::Type>
+emberAfTagListFromEndpoint(chip::EndpointId endpoint, CHIP_ERROR & err);
```

Comment:
Please do not create more `emberAf` APIs. Rather, create a `GetTagListFromEndpoint`.